examples/demo_runner.py

```diff
@@ -338,6 +339,14 @@ def init_common(self):
         random.seed(self._sim_settings["seed"])
         self._sim.seed(self._sim_settings["seed"])
 
+        recompute_navmesh = self._sim_settings.get("recompute_navmesh")
+        if recompute_navmesh or not self._sim.pathfinder.is_loaded:
```

Comment:
Minor comment: this does not allow turning recompute off explicitly if desired. 🤷‍♂️

Reply:
@aclegg3 I was thinking about adding an explict if === false check so that would have errored out instead, but decided against it. Thoughts?